packages/manager/apps/veeam-backup/src/pages/dashboard/Dashboard.page.tsx

```diff
@@ -18,7 +18,7 @@ import {
 } from '@ovh-ux/manager-module-vcd-api';
 import { Breadcrumb } from '@/components/Breadcrumb/Breadcrumb';
 import { urls } from '@/routes/routes.constant';
-import { SuccessMessages } from '@/components/Messages/SuccessMessage.component';
+import { MessagesViewer } from '@/components/Messages/Message.component';
```

Comment:
niptick : use the same name as your component
```suggestion
import { Message } from '@/components/Messages/Message.component';
```